apis/google/sheets/code/writeCellsToSheet.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from gspread.exceptions import APIError
+
+from auth import gspread_client
+
+
+def main():
+    spreadsheet_id = os.getenv('SPREADSHEET_ID')
+    spreadsheet_name = os.getenv('SPREADSHEET_NAME')
+    if spreadsheet_id is None and spreadsheet_name is None:
+        raise ValueError("Either spreadsheet_id or spreadsheet_name must be set")
+
+    raw_data = os.getenv('DATA')
+    if raw_data is None:
+        raise ValueError("data must be set")
+    else:
+        rows_data = raw_data.split('\n')
+        data = [row.split(',') for row in rows_data]
```

Comment:
This logic is pretty brittle. ive ran into issues with it multiple times now where content thats meant for a cell had commas in it.

Reply:
We discussed this out of band - I fixed this by running it through the python csv parser and telling the llm to format the data in such a way that it would be valid csv.